docs/tutorial/a_minimal_working_example.rst

```diff
@@ -0,0 +1,128 @@
+A minimal working example
+=========================
+
+We start with a minimal working example of ALF. The example, as a pure ALF
+configuration file, is located at ``<ALF_ROOT>/alf/examples/tutorial/minimal_example_conf.py``,
+and consists of only 8 lines.
+
+Train and play
```

Comment:
I am not sure `play` is a well understood terminology. If not it might worth explaining. For example: play is the procedure of applying the trained agent/policy to perform the task and possibly also visualize it.